hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java

```diff
@@ -767,4 +717,18 @@ protected NearCacheConfig newNoInvalidationNearCacheConfig() {
     protected ClientConfig newClientConfig() {
         return new ClientConfig();
     }
+
+    protected void assertNearCacheInvalidation_whenMaxSizeExceeded(NearCacheConfig config) {
+        final IMap<Integer, Integer> map = getNearCachedMapFromClient(config);
+        populateNearCache(map, MAX_CACHE_SIZE);
```

Comment:
the intention of test is to start invalidating entries after the cache is full, so you only need MAX_SIZE for the  cache to be full. The rewritten version fills the cache and then triggers eviction by inserting one more item.